test/inductor/test_inductor_freezing.py

```diff
@@ -364,6 +364,21 @@ def foo(mod, inp):
             mod_eager = mod(x)
             self.assertEqual(foo(mod, x), mod_eager)
 
+    def test_cpp_wrapper(self):
+        mod = ConvBN(3, 32, kernel_size=3, stride=2).eval().to(self.device)
+
+        x = torch.rand(3, 3, 32, 32).to(self.device)
+
+        @torch.compile(options={"cpp_wrapper": True})
+        def foo(mod, x):
+            return mod(x)
+
+        out_eager = mod(x)
+
+        with torch.no_grad():
```

Comment:
I added a test for cpp_wrapper but I"m not sure how I should exercise the export path. 
If I do something like [this](https://github.com/pytorch/pytorch/blob/main/test/cpp/aot_inductor/test.py#L27) do I need to do any cleanup ?

Reply:
To test Export+AOTInductor, you can use a benchmarking script with --export-aot-inductor. alexnet should just do it.

I am also adding a unit test file in my another PR. You can also add a test to that later.